src/app/plugins/scores.plugin.ts

```diff
@@ -11,299 +12,119 @@ export class ScoresPlugin extends Plugin {
+          competitor.team.name.toLowerCase() == teamName
+      )
+    );
 
-      //Push last embed
-      embedBuffer.push(embed);
-    } else {
-      embed.setDescription(matchups.join('\n').replace(/@/g, ' at '));
-      message.reply(embed);
-    }
-    embedBuffer.forEach((e) => {
-      message.channel.send(e);
-    });
-  }
-  private _getNewEmbed() {
-    const embed = new RichEmbed();
-    embed.setTitle('Current games available');
-    embed.setColor('#7289da');
-    return embed;
+    return targetGame[0]; //There should theoretically be match
   }
 
-  private _evaluateScores(a: number, b: number): number {
-    if (a == b) return 0; //Tied
-    return a > b ? 1 : -1;
+  private async _getGames(url: string): Promise<espn.Event[]> {
+    const response = (await this.container.httpService.get(url)).data;
+    const responseData: espn.Sample = (response as Object) as espn.Sample;
+    return responseData.events;
   }
 
-  private _sendHelp(message: IMessage) {
-    message.reply({
-      embed: {
-        title: 'Scores Plugin Help',
-        color: '7506394',
-        fields: [
-          { name: 'Supported Sports', value: 'NCAA NFL MLB NBA' },
-          {
-            name: 'Inputting Teams',
-            value:
-              'For NCAA: Use their most common name\nFor Professional Sports: Use their city only\n',
-          },
-          {
-            name: 'Examples',
-            value:
-              '!scores NCAA UCF\n!scores NCAA Florida State\n!scores NFL Seattle\n!scores MLB Atlanta',
-          },
-        ],
-      },
-    });
+  private _getIsVisitor(game: espn.Event, teamName: string): boolean {
+    const visitorTeam: espn.Team = game.competitions[0].competitors[1].team;
+    return (
+      visitorTeam.location.toLowerCase() === teamName ||
+      visitorTeam.abbreviation.toLowerCase() === teamName ||
+      visitorTeam.name.toLowerCase() === teamName
+    );
   }
 
-  private _tryUpcomingGame(embedBuffer: RichEmbed[], game: any, teamArg: string): boolean {
-    let teamsData = game.match(this._UPCOMING_REGEX) || [''];
-    const time = String(game.match(this._UPCOMING_TIME_REGEX) || '').split('%20'); //Convert Array to string
-
-    //Make sure data is valid
-    if (teamsData.join(' ') === '' || time.join(' ') === '') {
-      return false;
-    }
-
-    teamsData = teamsData[0].replace('=', '').split('%20'); //Trim and split
-    const matchup = teamsData.join(' ');
-
-    teamsData = matchup.split(' at '); //Eliminate 'at' and combine multi-word team names
-    const [visitorName, homeName] = teamsData;
-
-    //Check if neither team contains all search terms
-    if (
-      !(
-        this._strictlyContainsAllTokens(visitorName, teamArg) ||
-        this._strictlyContainsAllTokens(homeName, teamArg)
-      )
-    ) {
-      return false;
-    }
+  private async _sendEmbed(message: IMessage, game: espn.Event, isVisitor: boolean) {
+    const embed = this._createEmbed(game, isVisitor);
```

Comment:
this is a code smell. it doesnt make sense for `_sendEmbed` to receive some arguments and immediately pass them to another function. it also makes your code inherently more complex for others to read because its hard to differentiate what the difference between sendEmbed and createEmbed is